src/routes/safe/container/index.tsx

```diff
@@ -1,76 +1,87 @@
-    isOpen: false,
-    selectedToken: undefined,
-  },
-  showReceive: false,
-}
+export const BALANCES_TAB_BTN_TEST_ID = 'balances-tab-btn'
+export const SETTINGS_TAB_BTN_TEST_ID = 'settings-tab-btn'
+export const APPS_TAB_BTN_TEST_ID = 'apps-tab-btn'
+export const TRANSACTIONS_TAB_BTN_TEST_ID = 'transactions-tab-btn'
+export const ADDRESS_BOOK_TAB_BTN_TEST_ID = 'address-book-tab-btn'
+export const SAFE_VIEW_NAME_HEADING_TEST_ID = 'safe-name-heading'
+export const TRANSACTIONS_TAB_NEW_BTN_TEST_ID = 'transactions-tab-new-btn'
 
-const SafeView = (): React.ReactElement => {
-  const [state, setState] = useState(INITIAL_STATE)
-  const safeAddress = useSelector(safeParamAddressFromStateSelector)
+const Apps = React.lazy(() => import('../components/Apps'))
+const Settings = React.lazy(() => import('../components/Settings'))
+const Balances = React.lazy(() => import('../components/Balances'))
+const TxsTable = React.lazy(() => import('src/routes/safe/components/Transactions/TxsTable'))
+const AddressBookTable = React.lazy(() => import('src/routes/safe/components/AddressBook'))
 
-  useLoadSafe(safeAddress)
-  useSafeScheduledUpdates(safeAddress)
+//const useStyles = makeStyles(styles as any)
 
-  const onShow = (action) => () => {
-    setState((prevState) => ({
-      ...prevState,
-      [`show${action}`]: true,
-    }))
-  }
+const Layout = (props: RouteComponentProps) => {
+  //const classes = useStyles()
+  const { match } = props
 
-  const onHide = (action) => () => {
-    setState((prevState) => ({
-      ...prevState,
-      [`show${action}`]: false,
-    }))
+  const [modal, setModal] = useState({
+    isOpen: false,
+    title: null,
+    body: null,
+    footer: null,
+    onClose: null,
+  })
+
+  const safeAddress = useSelector(safeParamAddressFromStateSelector)
+  const provider = useSelector(providerNameSelector)
+  if (!safeAddress) {
+    return <NoSafe provider={provider} text="Safe not found" />
   }
 
-  const showSendFunds = (token) => {
-    setState((prevState) => ({
-      ...prevState,
-      sendFunds: {
-        isOpen: true,
-        selectedToken: token,
-      },
-    }))
+  const openGenericModal = (modalConfig) => {
+    setModal({ ...modalConfig, isOpen: true })
   }
 
-  const hideSendFunds = () => {
-    setState((prevState) => ({
-      ...prevState,
-      sendFunds: {
-        isOpen: false,
-        selectedToken: undefined,
-      },
-    }))
+  const closeGenericModal = () => {
+    if (modal.onClose) {
+      modal.onClose()
+    }
+
+    setModal({
+      isOpen: false,
+      title: null,
+      body: null,
+      footer: null,
+      onClose: null,
+    })
   }
-  const { sendFunds, showReceive } = state
 
   return (
-    <Page>
-      <Layout
-        hideSendFunds={hideSendFunds}
-        onHide={onHide}
-        onShow={onShow}
-        sendFunds={sendFunds}
-        showReceive={showReceive}
-        showSendFunds={showSendFunds}
-      />
-    </Page>
+    <>
+      {/* <TabsComponent /> */}
```

Comment:
Can `TabsComponent` be removed from here?

Reply:
sure